astropy/units/equivalencies.py

```diff
@@ -152,6 +153,9 @@ def spectral():
     )
 
 
+@deprecated_renamed_argument(
+    "factor", None, since="7.0", alternative="`wav` as a `Quantity`"
```

Comment:
I don't think we need formatting here?
```suggestion
    "factor", None, since="7.0", alternative="wav as a Quantity"
```

Reply:
This text will be displayed to the users as is, i.e. with literal backticks, but I still think it is better to use it to distinguish `wav` and `Quantity` from normal English words. An alternative would be to use `'` instead of `` ` ``.

Reply:
I think this is good as is!

<img src="https://github.com/astropy/astropy/assets/8949649/3055220a-19b6-4a26-9b1b-91c956bf974d" width="300" />

It correctly becomes links / parameter references in the docs.

Reply:
> It correctly becomes links / parameter references in the docs.

I don't think so. The red box is from a different `.. deprecated` directive.

Reply:
What about this?

```python
alternative="'wav' as a Quantity"
```

Reply:
@pllim. Good catch!